src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -526,6 +543,45 @@ abstract class DatabaseTest {
     return row.id!!.also { inserted.projectIds.add(it) }
   }
 
+  private var nextDeliverableNumber: Int = 1
+
+  protected fun insertDeliverable(
+      createdBy: UserId = currentUser().userId,
+      createdTime: Instant = Instant.EPOCH,
+      deliverableCategoryId: DeliverableCategory? = DeliverableCategory.FinancialViability,
+      deliverableTypeId: DeliverableType? = DeliverableType.Document,
```

Comment:
These arguments could be non-nullable since the columns are non-nullable in the database.